crates/iota-sdk/Cargo.toml

```diff
@@ -24,6 +24,7 @@ rustls.workspace = true
 serde.workspace = true
 serde_json.workspace = true
 serde_with.workspace = true
+signature.workspace = true
```

Comment:
Just commenting to mention that this dep was already in the tree from `iota-keys`, so this is just added so that we can directly use the types.